src/api/types/trip.ts

```diff
@@ -33,10 +33,10 @@ export interface ITripsQueryRequest {
     originId: string;
     destinationId: string;
     tripDate: string;
-    sortBy: ITripsSortKey;
-    sortOrder: ISortOrder;
-    page: number;
-    pageSize: number;
+    sortBy?: ITripsSortKey;
+    sortOrder?: ISortOrder;
+    page?: number;
```

Comment:
page and pagesize are not optional

Reply:
Page size and page changed to not optional: https://github.com/nijin-26/bus-ticket-booking-react/pull/24/commits/3e2d69f65aeb832dd70daaced22b1f27440f3297